addons/web/static/src/views/fields/text/text_field.xml

```diff
@@ -1,28 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-
     <t t-name="web.TextField" owl="1">
-        <t t-if="props.readonly">
-            <span t-esc="props.value or ''" />
-        </t>
-        <t t-else="">
-            <textarea
-                class="o_input"
-                t-att-class="{'o_field_translate': props.isTranslatable}"
-                t-att-id="props.id"
-                t-att-placeholder="props.placeholder"
-                t-att-rows="rowCount"
-                t-on-input="onInput"
-                t-ref="textarea"
-            />
-            <t t-if="props.isTranslatable">
-                <TranslationButton
-                    fieldName="props.name"
-                    record="props.record"
-                    updateField="props.update"
-                />
+            <t t-if="props.readonly">
+                <span t-esc="props.value or ''" />
+            </t>
+            <t t-else="">
+                <div class="o_field_text_wrapper">
```

Comment:
Button behaved a bit weird with negative margins so I resorted to absolute positioning, where I need some kind of reliable anchor point to be 'relative' (o_text_field_widget isn't reliable because that div gets extended when we add text to it in field_sms_widget).

I get the problem though. I'll try to make it work with negative margins and all again but it seems like it might be prone to breaking even if I can.